### PR TITLE
Sync stream creation

### PR DESCRIPTION
**Pull request by sevko**
Address #6 .

### CODE REVIEW

**Comment by sevko** [comment]
Merging when @dianashk and @missinglink sign off.

File: package.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
   "name": "pelias-admin-lookup",
-  "version": "1.0.1",
+  "version": "1.0.2",
```
Comment (dianashk) [inline package.json]: This isn't a big deal, but don't we usually up the version in a separate commit on master?

Comment (sevko) [inline package.json]: Yeah, this was my bad.

**Comment by dianashk** [comment]
+1 (other than package version)

**Comment by missinglink** [comment]
This is super awesome, the interface is much cleaner and easier to use, couple comments:

1. it would be best to defend against consumers instantiating 2x streams:

```javascript
adminLookup = require( 'pelias-admin-lookup' )
stream1 = adminLookup.stream();
stream2 = adminLookup.stream();
```

2. you should increment a change to the major version number (ie bump to v2.0.0) due to the breaking API change

**Pull request merged**